recipe/meta.yaml

```
@@ -19,16 +19,17 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.8
+    - python >=3.9
     - setuptools
 
   run:
-    - python >=3.8
-    - dask >=2021.10.0
-    - numpy >=1.11.3
+    - python >=3.9
```

Same question as above

```suggestion
    - python >=3.8
```